src/main/java/org/openstreetmap/atlas/locale/IsoCountryFuzzyMatcher.java

```diff
@@ -99,41 +65,46 @@
         return new ArrayList<>();
     }
 
+    /*
+     * This algorithm could definitely be improved for cases with multi-word country matching.
+     */
     private static List<String> closestIsoCountries(final int number, final String displayCountry)
     {
         if (number <= 0 || number > IsoCountry.ALL_DISPLAY_COUNTRIES.size())
         {
             throw new CoreException("number " + number + " out of range (0, "
-                    + IsoCountry.ALL_DISPLAY_COUNTRIES.size() + ")");
+                    + IsoCountry.ALL_DISPLAY_COUNTRIES.size() + "]");
```

Comment:
Is that intended?

Reply:
Yes! I was trying to make it comply with this kind of notation: https://en.wikipedia.org/wiki/Bracket_(mathematics)#Intervals